frontend/workflows/ec2/src/terminate-instance.tsx

```diff
@@ -31,19 +31,25 @@ const InstanceDetails: React.FC<WizardChild> = () => {
   const resourceData = useDataLayout("resourceData");
   const instance = resourceData.displayValue();
 
+  const data = [
+    { name: "Instance ID", value: instance.instanceId },
+    { name: "Region", value: instance.region },
+    { name: "State", value: instance.state },
+    { name: "Instance Type", value: instance.instanceType },
+    { name: "Public IP Address", value: instance.publicIpAddress },
+    { name: "Private IP Address", value: instance.privateIpAddress },
+    { name: "Availability Zone", value: instance.availabilityZone },
+  ];
+
+  if (instance.tags) {
+    Object.keys(instance.tags).forEach(key => {
+      data.push({ name: key, value: instance.tags[key] });
```

Comment:
Should we ensure title casing of the name?

Reply:
i don't think so. tags are a case-sensitive field on the server and should be presented directly to the user.

Reply:
however we probably want to differentiate between tags and regular informational fields. i could have a tag which is Region: iad, then you would get two rows in the table with the same header and diff value.

Reply:
@danielhochman what if we prepended the name with `tag: {key}` to indicate it is a tag?

Reply:
i like that or
![Screenshot from 2020-07-23 14-53-37](https://user-images.githubusercontent.com/4712430/88332529-54804380-ccf4-11ea-874e-de301ac249eb.png)


a separate table with its own header could work too.

let's do the easy thing now (text label or icon prepended) and we can ask our UX person soon what the ideal solution would be and follow up.

Reply:
Sorry all. I merged this too early. @mcutalo88 feel free to open a follow up :)